xonsh/environ.py

```diff
@@ -1432,6 +1432,13 @@ def DEFAULT_VARS():
             "branch computations. This is a timeout per subprocess call, so the "
             "total time to compute will be larger than this in many cases.",
         ),
+        "VC_GIT_INCLUDE_UNTRACKED": Var(
+            is_bool,
+            to_bool,
+            bool_to_str,
+            False,
```

Comment:
can we have this default as True here? to retain the current behaviour

Reply:
hm good point. i thought it was `False` by default? just looking at the original `git_dirty_working_directory` args, btw

Reply:
Yeah, the existing default is to not include untracked files